mobile/src/main/res/values/dimens.xml

```diff
@@ -6,6 +6,7 @@
     <dimen name="widget_margin">0dp</dimen>
     <dimen name="row_height">36dp</dimen>
     <dimen name="widgetlist_icon_size">40dp</dimen>
+    <dimen name="widgetlist_compact_slider_width">100dp</dimen>
```

Comment:
TODO: make this shorter

Reply:
Actually this value wasn't the issue, but a `layout_weight` being assigned to the slider. Fixed now.